charts/flyte-core/values.yaml

```diff
@@ -60,6 +59,8 @@ flyteadmin:
   secrets: {}
   additionalVolumes: []
   additionalVolumeMounts: []
+  # -- Deploys a Redoc container in Flyteadmin's pod
+  deployRedoc: true
```

Comment:
should we maybe always have this set to false and call out in the docs how and why you would want to enable this?

Reply:
Good point about the docs. I'll do that... I only kept it on for backward compatibility...